collect_app/src/main/res/values/strings.xml

```diff
@@ -340,7 +340,6 @@
     <string name="select_geotrace_mode">"Select GeoTrace Mode"</string>
     <string name="start">Start</string>
     <string name="enable_gps">Enable GPS</string>
-    <string name="enable_gps_message">"GPS is disabled in your device. Would you like to enable it?"</string>
```

Comment:
Took me a bit to figure out that `enable_gps_message` was redundant to `gps_enable_message` and that it's staying. 😳

Reply:
Sorry, I probably should have explained what was happening there!  A couple of duplicate string references went away, so the linter detected that and required removal of the corresponding string resources.

Reply:
Good to see the duplicates gone.